browser/ui/views/tabs/brave_tab_container.cc

```diff
@@ -294,6 +333,14 @@ void BraveTabContainer::PaintChildren(const views::PaintInfo& paint_info) {
 
   std::stable_sort(orderable_children.begin(), orderable_children.end());
 
+  if (base::FeatureList::IsEnabled(tabs::features::kBraveSplitView)) {
+    ui::PaintRecorder recorder(paint_info.context(),
```

Comment:
Q) I don't know about PaintRecorder. Is it used to get canvas easily?

Reply:
Yes, it's used to record paint ops along with view tree. It's also used in views::View.